Demo_Project/src/Component/settings/Settings.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
```

Comment:
why all `TextField`s don't have `onChange` handlers?

Reply:
up to now i just doing styling all these function i am going to do it when start doing functionality